<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="player_south">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="player_south_point">
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="southPB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="south_richi">
         <property name="text">
          <string>Richi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="player_west">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="player_west_point">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="westPB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="west_richi">
         <property name="text">
          <string>Richi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="westplayer">
       <item>
        <widget class="QLineEdit" name="player_north">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="player_north_point">
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="northPB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="north_richi">
         <property name="text">
          <string>Richi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="player_east">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="player_east_point">
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eastPB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="east_richi">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Richi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="current_oya"/>
         </item>
         <item>
          <widget class="QSpinBox" name="round_count"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QSpinBox" name="pon"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>player_north</tabstop>
  <tabstop>westPB</tabstop>
  <tabstop>northPB</tabstop>
  <tabstop>player_south</tabstop>
  <tabstop>southPB</tabstop>
  <tabstop>player_west</tabstop>
  <tabstop>player_east</tabstop>
  <tabstop>eastPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
